tests/integration/Cake.Core/Scripting/AddinDirective.cake

```diff
@@ -11,7 +11,13 @@ Task("Cake.Core.Scripting.AddinDirective.LoadNetStandardAddin")
         Information(""The answer to life: {0}"", TheAnswerToLife);
     ";
 
-    CakeExecuteExpression(script);
+    CakeExecuteExpression(script,
+    	new CakeSettings {
+    		EnvironmentVariables = new Dictionary<string, string>{
+    			{"CAKE_PATHS_ADDINS", $"{Paths.Temp}/tools/Addins"},
```

Comment:
@mholo65 this was changed because saw an issue on multiple Windows systems with addin installation as Cake changes current directory to script dir, which for expression is Windows temp director, setting it to be in a known directory within integration tests sorted it.

Long term CakeExecuteExpression probably should set these by default if not already set to the paths in current cake process config as that's probably the expected behaviour. 

Unreleated to issue so could remove, just couldn't run integration tests locally without it atm.